### ISSUE TITLE
Add an option to fail builds on warnings

### ISSUE THREAD

**Issue by YANOUSHek** [sleekbyte/tailor#476]
This is not exactly an issue but a missing feature. I'd like to set up my Xcode so that any message from tailor fails the build (even a warning). Can't find an option to do this.

Can we add this?

**Comment by adityatrivedi** [comment]
@YANOUSHek, is [this](https://github.com/sleekbyte/tailor#warnings-errors-and-failing-the-build) what you are looking for?

**Comment by YANOUSHek** [comment]
@adityatrivedi nope, this does not allow to fail the build when there's only a warning detected by tailor.

> Setting it to error [...] also fails the build in Xcode, if any errors are reported

> With max-severity set to warning, all violation messages are warnings and the Xcode build will never fail.

There's no option to make any message (no matter if it's a warning or error) fail the Xcode build.

**Comment by adityatrivedi** [comment]
Got it, thanks for clarifying the enhancement you’re looking for. Ideally Xcode’s “treat warnings as errors” would allow you to do exactly that, but it seems to only work for compiler warnings and not Build Phase Run Script output. We envision this being a separate command-line option. Our team is pretty time constrained at the moment, a pull request to add a `--strict` flag would be welcome.
